linera-chain/src/data_types.rs

```diff
@@ -1266,8 +1311,8 @@ impl Certificate {
 }
 
 /// Verifies certificate signatures.
-fn check_signatures(
-    value: &LiteValue,
+pub(crate) fn check_signatures(
```

Comment:
Does this still need to be `pub(crate)`?

Reply:
It can be either `pub` or `pub(crate)` - i.e. it cannot be private (at least not right now).

Reply:
Where is it used? I honestly can't find it. :sweat_smile:

Reply:
In `Certificate` and `LiteCertificate` `check` methods.

Reply:
Exactly: Aren't they in the same module, so that this can be private again?

Reply:
Ah, you're right. I was checking this on the other branch (now a draft PR that adds separate types for blocks) where I'm reusing it in a different file 🙈 I can make this private again but will add `pub(crate)` in the other PR.

Reply:
Done in https://github.com/linera-io/linera-protocol/pull/2745/commits/88a8ed2795ca07a09915a172e32bb7dd0bce1052.